docs/index.md

```diff
@@ -42,6 +42,7 @@ https://retroachievements.org/API/API_GetAchievementOfTheWeek.php?z=[your_userna
 
 ## Client Libraries
 
-| Platform / Language | Repo                                                                      |
-| :------------------ | :------------------------------------------------------------------------ |
-| NodeJS (JavaScript) | [`RetroAchievements/api-js`](https://github.com/RetroAchievements/api-js) |
+| Platform / Language | Repo                                                                              |
+| :------------------ | :-------------------------------------------------------------------------------- |
+| NodeJS (JavaScript) | [`RetroAchievements/api-js`](https://github.com/RetroAchievements/api-js)         |
+| Kotlin              | [`RetroAchievements/api-kotlin`](https://github.com/RetroAchievements/api-kotlin) |
```

Comment:
I'm not very well-versed in Kotlin. I was hoping I could get your opinion here.

Do you think it makes sense for this to be listed as "JVM (Kotlin)"? I'm wondering if this may be particularly relevant if the Kotlin lib can be used in Java, Scala, or Groovy projects.